src/libraries/Common/src/System/Runtime/InteropServices/ComEventsSink.cs

```diff
@@ -10,7 +12,7 @@ namespace System.Runtime.InteropServices
     /// Part of ComEventHelpers APIs which allow binding
     /// managed delegates to COM's connection point based events.
     /// </summary>
-    internal class ComEventsSink : IDispatch, ICustomQueryInterface
+    internal partial class ComEventsSink : IDispatch, ICustomQueryInterface
```

Comment:
>IDispatch, ICustomQueryInterface [](start = 43, length = 32)

Nice! In olden time this had to use `IReflect/RealProxy` stuff to pretend it implements interfaces.